### PR TITLE
windows: use quotes around copy operation parameters

### PR DESCRIPTION
**Pull request by jupe**
Allows to flash files and targets with spaces

### CODE REVIEW

File: test/test_flash.py [hunk 1]
```diff
@@ -125,6 +125,15 @@ def test_run_with_uppercase_HTM(self, mock_popen, mock_verifier, mock_copy_file)
         self.assertEqual(ret, 0)
         self.assertEqual(2, mock_out.call_count)
 
+    @unittest.skipIf(platform.system() != 'Windows', 'require windows')
+    @mock.patch('os.system')
+    def test_copy_file_with_spaces(self, mock_system):
+        flasher=FlasherMbed()
+        srcfile = "%s/test_flash.py" % os.path.abspath(os.curdir)
```
Comment (YifeiZuo01) [inline test/test_flash.py]: os.curdir seems is `mbed-flasher/`, should be `mbed-flasher/test`

Comment (YifeiZuo01) [reply test/test_flash.py]: and `/` might be a problem

Comment (juhhov) [reply test/test_flash.py]: @YifeiZuo01 is this now ok?

Comment (YifeiZuo01) [reply test/test_flash.py]: yes.

**Pull request merged**